docs/FeatureManagement/Samples/TargetingFilter.modified.tests.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-Aiden62",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, 62% default rollout Aiden is part of it."
+    },
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-AidenStage162",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden", "groups":["Stage1"]},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, group is not part of default rollout calculation, no change."
+    },
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-AidenStage262",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden", "groups":["Stage2"]},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, group is not part of default rollout calculation, no change."
+    },
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-AidenStage362",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden", "groups":["Stage3"]},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, group is not part of default rollout calculation, no change."
+    },
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-Brittney62",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Brittney"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, 62% default rollout Brittney is not part of it."
```

Comment:
If Brittney is not part of it, shall we return "false" as the expected result?

Reply:
Copy Paste issue from the other description..